espresso-support/core/src/main/java/com/novoda/espresso/ViewTestRule.java

```diff
@@ -25,11 +27,14 @@ protected Intent getActivityIntent() {
     }
 
     public void bindViewUsing(final Binder<T> binder) {
+        isIdle = false;
         getActivity().runOnUiThread(new Runnable() {
             @Override
             public void run() {
                 T view = getView();
                 binder.bind(view);
+                callback.onTransitionToIdle();
```

Comment:
Seems useful if you have multiple jobs occurring at the same time, we know that it will always be one so not sure if super useful atm...